hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/AutoCreatedQueueTemplate.java

```diff
@@ -155,14 +155,13 @@ public void setTemplateEntriesForChild(CapacitySchedulerConfiguration conf,
    * yarn.scheduler.capacity.root.*.auto-queue-creation-v2.template.capacity
    */
   private void setTemplateConfigEntries(CapacitySchedulerConfiguration configuration,
```

Comment:
I think this could be also refactored in a follow-up jira so the string magic could probably be replaced with some more elegant solution. Though, I think this would be too much in this patch, hence I do suggest the follow-up jira.

Reply:
@brumi1024 @9uapaw Thoughts?

Reply:
+1, even the QueuePath object could have some kind of support for this.

Reply:
Agreed, let's handle it in a followup!

Reply:
A followup Jira is created: https://issues.apache.org/jira/browse/YARN-11041